src/types.js

```diff
@@ -1,22 +1,9 @@
 // @flow
 import type {ErrorCode} from "./error-codes.js";
 
-// const exampleMarkersA = {
-//     filea: {
-//         markerA: {
-//             fixable: true,
-//             checksum: "345678", // The actual checksum
-//             targets: [
-//                 { line: 10, file: "fileb", checksum: "2345678" /* What this file thinks this checksum should be for this marker */  },
-//                 { line: 104, file: "filec", checksum: "2345678" },
-//             ],
-//         },
-//     },
-// };
```

Comment:
This is wrong now and sketching out the new format felt laborious, so I just deleted it.